test/Microsoft.AspNet.Mvc.FunctionalTests/JsonPatchTest.cs

```diff
@@ -49,6 +50,119 @@ public class JsonPatchTest
         }
 
         [Theory]
+        [InlineData("http://localhost/jsonpatch/JsonPatchWithoutModelState")]
+        [InlineData("http://localhost/jsonpatch/JsonPatchWithModelState")]
+        [InlineData("http://localhost/jsonpatch/JsonPatchWithModelStateAndPrefix?prefix=Patch")]
+        public async Task JsonPatch_ValidReplaceOperation_Success(string url)
+        {
+            // Arrange
+            var server = TestHelper.CreateServer(_app, SiteName, _configureServices);
+            var client = server.CreateClient();
+
+            var input = "[{ \"op\": \"replace\", \"path\": \"Customer/Orders/0/OrderName\", " +
+               "\"value\": \"ReplacedOrder\" }]";
```

Comment:
spacing these out so each key-value is on it's own line would make this much easier to read.